<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор интерфейса</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTableView{
	background-color:#ffffff;
	alternate-background-color:#e2e2e2;
	color:#3a3b3a;
	border-style:solid;
	border-color:#ffffff;
	border-width:0px;
	selection-color:#ffffff;
	selection-background-color:#54b0cb;
}
QHeaderView::section{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:40px;
	border: #ffffff;
	border-style:solid;
	border-width:0px;
}
QComboBox{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:50px;
	height:30px;
	font:bold;
	padding-left:5px
}
QComboBox QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;
    border: 2px solid #ffffff;
	padding-left:2px;
	padding-right:2px;
	padding-top:2px;
	padding-bottom:2px
}
QComboBox::drop-down {
   width: 24px;
   border: 2px;
}
QComboBox::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:150px;
	height:30px;
	font:bold;
	padding-left:5px
}
QDateEdit QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;;
}
QDateEdit::drop-down {
   width: 24px;
   border: 2px;
}
QDateEdit::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QMenuBar{
	background-color:#3a3b3a;
	color:#ffffff;
	selection-color:#54b0cb;
	font:bold
}
QMenuBar::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	padding-left:2px
}
QToolBar{
	background-color:#3a3b3a;
	color:#ffffff;
}
QMenu {
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	padding-left:2px;
}
QMenu::hover {
    background-color: #448fa6;
    color:#3a3b3a;
	font:bold
}
QMenu::item {
    background-color:#3a3b3a;
	color:#ffffff;
}
QMenu::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	padding-left:10px;
}
QTabWidget{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTabBar::tab:right:selected{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QTabBar::tab:right:!selected{
    background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QLabel{
	background-color:#448fa6;
	color:#ffffff;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QGroupBox{
	background-color:#448fa6;
	color:#ffffff;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QPushButton{
	background-color:#3a3b3a;
	color:#ffffff;
	height:60px;
	width: 40px;
	border: #ffffff;
	border-style:solid;
	border-width:5px;
	font:14px;
	font-weight: bold
}
QPushButton::pressed{
	background-color:#ffffff;
	color:#3a3b3a;
	border:2px solid #3a3b3a;
}
QPushButton::hover{
	color:#448fa6;
	border:2px solid #448fa6;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Для чего будет использоваться этот компьютер?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="arrivalButton">
         <property name="text">
          <string>Терминал точки прибытия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scaleButton">
         <property name="text">
          <string>Весовой терминал</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unloadButton">
         <property name="text">
          <string>Терминал пункта выгрузки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="operatorButton">
         <property name="text">
          <string>Рабочее место оператора приемки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="laboratoryButton">
         <property name="text">
          <string>Рабочее место лаборанта</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
